### ISSUE TITLE
Smart image sizing and loading for key stories and footer

### ISSUE THREAD

**Issue by jread-usgs** [USGS-VIZLAB/water-use-15#313]
this is going to take over for #304 

We have footer images and water use stories that are 1.3 MB (desktop story), 757 KB (mobile story right), 574 KB (mobile story), 117 KB (the previous WU viz's thumbnail in the footer), and 47.3 KB (Maria thumbnail in the footer). 

For reference, D3 is 72.8 KB and county_boundaries_USA.json are 242 KB

The three WU story jpegs are the three biggest files the viz loads initially (high res counties are big too, but they aren't loaded until you zoom). 

These three images seem to get loaded before we start loading the state data or the county centroids, and therefore they are likely slowing the "time to first paint". The implications are approximately a > 1.5s delay on network for a desktop machine and a much larger delay on mobile cell data. 

 - [ ] make it so either the single desktop image is used/loaded (wide) or two (narrow) for water use
 - [ ] shrink the two mobile files (they are likely higher resolution than is needed for the narrow view
 - [ ] modify the way these resources are fetched by the browser so that they are queued later, either by implementing some lazy loading solution (there are many options including starting w/ a lower-res/blurry image and replacing w/ higher res when it is in viewport) or just making sure they are called for _after_ the map is built.

**Comment by jread-usgs** [comment]
Looking good as of #326. Now that we have a single "wide" image for mobile, we might want to do some work to shrink the size/resolution of that image. The one in right now https://beta-owi.usgs.gov/vizlab/water-use-15/images/WU_stories_static_mobile.jpg is 1747px wide, and it looks like we switch to using this image when the resolution of the device is 600px wide. 

Also, since loading the footer images delays drawing the map, I think we should also use the lazy load for those images.

**Comment by aappling-usgs** [comment]
Are there supposed to be two `(4)`s in that image?

**Comment by jread-usgs** [comment]
@lindsaycarr ☝️ 
I think I remember you having a key for each of those numbers in the mobile version ("read more"). Is 4 supposed to repeat?

**Comment by mwernimont** [comment]
So the 4 is from the same blurb we broke up, so it's talking about the same thing, in two different areas of the country. We could do 4.1 and 4.2 if you guys find it confusing?  We do have keys expander and hider coming up that explains the numbers in greater detail.

**Comment by lindsaycarr** [comment]
So I see we fixed the issue with two 4's above by having 4a and 4b, and the footer images are indeed lazy-loading.

Looks like there one thing left: shrink width of that mobile image to 600px. @mwernimont do you see any reason not to do that, and if not, is that something you can do?

**Comment by mwernimont** [comment]
Is it just a file size issue?  Cause we already control how big the image is through CSS.  If this is cause it switches at 600px I would start to worry about resolution and text size.

**Comment by jread-usgs** [comment]
Yes, this is a file size issue. I am assuming we can get a smaller file size and not lose quality for the < 600px devices. Is that not the case?

**Comment by mwernimont** [comment]
No, it is, I was blissfully unaware of the current 1700px+ image size for mobile.  It will be shrunk when I get the new and improved base map form Lindsay
